examples/node/package.json

```diff
@@ -7,11 +7,19 @@
   "author": "Max Klenk <max@li.finance>",
   "license": "MIT",
   "dependencies": {
-    "@lifi/sdk": "latest"
+    "@lifi/rpc-wrapper": "^0.0.23",
+    "@types/dotenv": "^8.2.0",
+    "dotenv": "^16.1.1"
+  },
+  "scripts": {
+    "start": "ts-node index.ts"
   },
   "devDependencies": {
     "@types/node": "^18.11.17",
     "ts-node": "^10.8.2",
     "typescript": "^4.9.4"
+  },
+  "resolutions": {
+    "@lifi/sdk": "portal:/Users/abhishekkumar/Desktop/work/sdk"
   }
```

Comment:
Please remove this once you finish testing 🙂

Reply:
Done, removed it.